.github/workflows/run-smokes-manual.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  workflow_dispatch:
```

Comment:
__Nit__: Shouldn't we just add `workflow_dispatch` to `smoke-test.yml`?

Reply:
So I tried that initially, but the logic in there gets really complicated because GitHub doesn't have a great way of saying "calculate this for me but if I give you an override ignore it" without very messy logic. This felt like the better compromise.